src/query/service/src/interpreters/interpreter_merge_into.rs

```diff
@@ -51,6 +53,8 @@ use crate::pipelines::PipelineBuildResult;
 use crate::schedulers::build_query_pipeline_without_render_result_set;
 use crate::sessions::QueryContext;
 
+// predicate_index should not be conflict with update expr's column_binding's index.
+pub const PREDICATE_COLUMN_INDEX: IndexType = MAX;
```

Comment:
does this conflict with https://github.com/datafuselabs/databend/blob/0afc3f3eb52daea8794bca1fe9b1506a6a8bed37/src/query/expression/src/schema.rs#L49

Reply:
no, it's just a tag to find column. it's use for columnBounding's index, please just this think it's a name.

Reply:
@ariesdevil wait, it's riskful.

Reply:
it's safe, in metadata, the row_id_col idx is retrived by metatda.columns.len().